src/main/java/seedu/dengue/logic/commands/DeleteCommand.java

```diff
@@ -65,7 +65,7 @@ public DeleteCommand(List<Index> targetIndexes) {
      * @param targetIndex
      */
     public DeleteCommand(Index targetIndex) {
-        this.targetIndexes = Optional.of(List.<Index>of(targetIndex));
+        this.targetIndexes = Optional.of(List.of(targetIndex));
```

Comment:
Why remove generic type? I think it's safer to let it remain. Can use generics on the optional as well.

Reply:
It can be inferred, I think. IntelliJ is flagging it out as redundant haha. I'll add it back in a later commit!